source/_drafts/Video-js-6-0-0-RC-0-The-first-RC.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Video.js 6.0.0-RC.0: The first Release Candidate'
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+  - video.js 6.0
+---
+
+## Release Candiate Released
+Last week, we published the first Release Candidate (RC) of Video.js 6.0 on npm.
+This was a months-long team effort to make Video.js even better.
+However, to do so, we had to make a few breaking changes.
+
+## What to look forward to
+There are several big things that changed in Video.js 6.0.
+* We're finally removing flash from core as outlines in [a previous post](http://blog.videojs.com/the-end-of-html-first/).
+* Plugins are being updated to a React-inspired component architecture. (The old style is staying around).
+* We're recommitting to accessiblity by fixing the accessibilty of our volume control and bringing back outlines!
+* Middleware. A brand new feature to interface between Video.js's techs and the player.
+
+## Feature Spotlights
+Over the coming few weeks, wee'll post feature spotlights talking about the big new things that are happening.
```

Comment:
"Over the coming ~~few~~ weeks, ~~wee'll~~ we'll post feature spotlights talking about the big ~~new~~ things that are happening